runtime/shiden/src/xcm_config.rs

```diff
@@ -161,6 +161,7 @@ impl SafeCallFilter {
             RuntimeCall::System(..)
             | RuntimeCall::Identity(..)
             | RuntimeCall::Balances(..)
+            | RuntimeCall::Utility(pallet_utility::Call::as_derivative { .. })
```

Comment:
Isn't this already covered on line 200?

Reply:
Sorry, I didn't test it, just observed the problem

<img width="924" alt="image" src="https://github.com/AstarNetwork/Astar/assets/57781136/b1fa9f45-069b-4e57-8b9b-43ba80e63bd8">

Reply:
The purpose of my submission of this pr is to hope that the astar network can support as_derivative, so I only modify the xcm_config file under astar, and I have revoked the modification of shiden

Reply:
@hqwangningbo did you check line 220 as I asked in previous comment?

About the part which you highlighted returns `false`, what exact type of call do you want to achieve?
Such information should be in the PR summary btw.